easybuild/easyblocks/b/boost.py

```diff
@@ -115,15 +116,55 @@ def prepare_step(self, *args, **kwargs):
     def configure_step(self):
         """Configure Boost build using custom tools"""
 
-        # mpi sanity check
-        if self.cfg['boost_mpi'] and not self.toolchain.options.get('usempi', None):
-            raise EasyBuildError("When enabling building boost_mpi, also enable the 'usempi' toolchain option.")
+        # Disable any Boost features in the configuration (bootstrap) step, it won't be possible at later stages
+        bootstrap_opts = [opt.split('=') for opt in self.cfg['configopts'].split()]
+        bootstrap_bools = [opt[0] for opt in bootstrap_opts if len(opt) == 1]  # safekeep all boolean opts
+        bootstrap_opts = {opt[0]: opt[1].split(',') for opt in bootstrap_opts if len(opt) > 1}  # take key/value pairs
```

Comment:
@lexming This syntax is not compatible with Python 2.6...

Reply:
undusted the docs of Python 2.6 and fixed